templates/output_params_txt.xml

```diff
@@ -7,15 +7,35 @@
            <option value="true">{{ _("alternate Z value") }}</option>
            <option value="false">{{ _("Z travel per stitch") }}</option>
         </param>
-        <param name="stitch_z_travel" type="float" gui-text="{{ _("Z travel per stitch") }}" gui-description="{{ _('increment z coordinate by this amount per stitch if "Z travel per stitch" is enabled') }}">5.0</param>
+        <param name="stitch_z_travel" type="float" gui-text="{{ _("Z travel per stitch") }}"
+            gui-description="{{ _('increment z coordinate by this amount per stitch if "Z travel per stitch" is enabled') }}">5.0</param>
```

Comment:
The best part about this is that Jinja automatically converts the `"` to `&#34;`!!